CvGameCoreDLL/CvCity.cpp

```diff
@@ -4260,6 +4262,31 @@ void CvCity::processBonusEffect(BonusTypes eBonus, int iChange)
 }
 
 
+// Leoreth
+void CvCity::processBonusEffect(BonusTypes eBonus, int iChange)

```

Comment:
Looks like the diff is messed up here, can you explain in more detail what changes you made? Preferably by annotating the corresponding lines.